src/kaleidoscope/key_defs.h

```diff
@@ -255,14 +241,15 @@ typedef kaleidoscope::Key Key_;
 #define KEY_LEFT_FN2 0xff
 #define Key_LFN2 Key(KEY_LEFT_FN2, KEY_FLAGS)
 
+#define SYSTEM_KEY(code) Key(code, SYNTHETIC | IS_SYSCTL)
```

Comment:
To simplify the definition of System Control keys, I added this macro as the counterpart to the existing `CONSUMER_KEY()` macro below.